src/components/dashboard/constants.js

```diff
@@ -2,7 +2,7 @@
 export const SIDE_MENU_ITEMS = [
     { iconType: 'dashboard', label: 'Overview', link: '/dashboard' },
     { iconType: 'sound', label: 'Campaigns', link: 'dashboard/campaigns' },
-    { iconType: 'environment', label: 'Locations' },
+    { iconType: 'environment', label: 'Locations', link: 'dashboard/locations' },
     { iconType: 'team', label: 'Users' },
     { iconType: 'user', label: 'Staff' },
```

Comment:
The `link` in each array item should be a constant. It makes it easy to update if you change the routes